### PR TITLE
County Priority - Ready

### PR DESCRIPTION
**Pull request by Gr770**

### CODE REVIEW

File: common/flavorization/00_CTP.txt [hunk 1]
```diff
@@ -1283,7 +1283,7 @@ count_tribal_female_anglo_saxon = {
 	gender = male
 	special = holder
 	tier = county
-	priority = 18
+	priority = 30
```
Comment (Charanraja123) [inline common/flavorization/00_CTP.txt:1286]: Should be 20

File: common/flavorization/00_CTP.txt [hunk 2]
```diff
@@ -5361,7 +5361,7 @@ count_tribal_female_north_germanic_group_independent = {
 	gender = female
 	special = holder
 	tier = county
-	priority = 19
+	priority = 32
```
Comment (Charanraja123) [inline common/flavorization/00_CTP.txt:5364]: what happened here?

**Review by Charanraja123 — CHANGES_REQUESTED** [review]
Overall, I think I have no problems raising the base - which as per you calculations is 19 technically (Since we add one for culture group anyway). 

In Any case, My Main Issue is that, culture and cultural groups have the same priority in your rework. Either we lower cultural group by one or increase cultures by 1. They should not be the same for obvious reason

File: common/flavorization/00_CTP.txt (hunk 2 above)
Comment (Gr770) [reply common/flavorization/00_CTP.txt:5364]: should be 21, i had my hand 1 over

**Comment by Charanraja123** [comment]
I've made some slight changes, for it to differentiate between culture groups and cultures. also to be in line with the rules we used.

**Comment by Charanraja123** [comment]
I've changed stuff, and Fixed until the Brythonic Culture Group.

There are two errors. All Barons got 153 instead of 103 (My fault during find and replace). 
Independent counts got 113 instead of 114. 

Rest of them Seem fine. So priority change is almost done.

**Review by Charanraja123 — APPROVED** [review]

**Pull request merged**